browserstack.yml

```diff
@@ -13,7 +13,7 @@ accessKey: YOUR_ACCESS_KEY
 # Set 'projectName' to the name of your project. Example, Marketing Website
 projectName: BrowserStack Samples 
 # Set `buildName` as the name of the job / testsuite being run
-buildName: browserstack build 
+buildName: test pytest
```

Comment:
why are we changing this?

Reply:
It was already in my branch, for a different use case. reverted this.